spec/controllers/omniauth_callbacks_controller_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+RSpec.describe OmniauthCallbacksController, :type => :controller do
+  before(:each) do
+    request.env["devise.mapping"] = Devise.mappings[:user]
+  end
+
+  describe "#facebook" do
+    it "should successfully authenticate using Facebook authentication hash and display a success message in flash" do
+      request.env["omniauth.auth"] = FactoryGirl.create(:auth_hash, :facebook)
+      get :facebook
+      user = User.find_by(email: 'testuser@facebook.com')
```

Comment:
You're no longer testing whether a user was properly added to the DB, which is probably something we want to verify. My comment earlier wasn't meant to imply that you should scrap that line but just that it needs to be rewritten. Since `find_by` will return `nil` when it doesn't find a match, `expect(user).not_to eq(nil)` should work here.